src/transport/net.cc

```diff
@@ -94,6 +96,8 @@ struct connectMapMem{
   char* cpuPtr;
   int size;
   ncclIpcDesc ipcDesc;
+  ncclShmIpcDesc_t attachDesc;
+  ncclShmIpcDesc_t createDesc;
   char shmPath[PATH_MAX];
```

Comment:
`char shmPath[PATH_MAX];` etc. no longer exists in latest NCCL as well. We need remove it.

Reply:
Solved in latest commit.